autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>apply <code>isort</code> to this cell</p><p> </p><p>do you want to pull the <code>pip install</code>  to above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
yes, and done